### ISSUE TITLE
Class not found.

### ISSUE THREAD

**Issue by james10174** [eclipse-che/che#962]
Trying to get server/client setup with Vaadin working with debugger in che. From what I have read, I have to create a connector component in a separate class the connect to main class. If there are any existing projects that work with Vaadin and debugger please let me know as it might be easier than trying to get this to work. Thanks.

FYI. I have been able to get the debugger to connect but just trying to figure out how to make server callbacks as everything seems to be executed in client right now.

![image](https://cloud.githubusercontent.com/assets/16562450/14253303/9cd87466-fa50-11e5-9230-b531cb24edb4.png)

![image](https://cloud.githubusercontent.com/assets/16562450/14253648/f6622904-fa51-11e5-8b99-25aeb8382b38.png)

![image](https://cloud.githubusercontent.com/assets/16562450/14253746/529ce34e-fa52-11e5-9b12-5d2dfbeb4b59.png)

**Comment by ghost** [comment]
You may take a look at this project http://beta.codenvy.com/f?name=Vaadin&user=ddementieva@codenvy.com

As to errors in your case, I can see multiple problems. 

First, can you go to Project > Configuration and configure this project as Maven?

Secondly, you may want to import this class as per https://vaadin.com/wiki/-/wiki/Main/Creating%20a%20simple%20component

**Comment by james10174** [comment]
The first link I get the following:

![image](https://cloud.githubusercontent.com/assets/16562450/14263862/b1ae21e2-fa82-11e5-90b0-ef1ae28100a8.png)
Might be that I have not received invite to codenvy beta.

Second link looks like a good example. It includes package name prefix. Might be problem. I will try it out and let you know.

**Comment by james10174** [comment]
Got it working! I will trying to post my code to github once I clean it up. The examples and documents are kind of hard to follow. At least for me it was.
